int/api/server.go

```diff
@@ -18,31 +17,28 @@ import (
 	pluginmanager "github.com/massalabs/thyra/pkg/plugins"
 )
 
-func parseFlags(server *restapi.Server) {
-	const httpPort = 80
-
-	const httpsPort = 443
-
-	flag.IntVar(&server.Port, "http-port", httpPort, "HTTP port to listen to")
-
-	flag.IntVar(&server.TLSPort, "https-port", httpsPort, "HTTPS port to listen to")
-
-	certFilePtr := flag.String("tls-certificate", "", "path to certificate file")
-	keyFilePtr := flag.String("tls-key", "", "path to key file")
-	massaNodeServerPtr := flag.String("node-server", "TESTNET", `Massa node that Thyra connects to. 
-	Can be an IP address, a URL or one of the following values: 'TESTNET', 'LABNET', 'INNONET' or LOCALHOST`)
+type StartServerFlags struct {
+	Port              int
+	TLSPort           int
+	TLSCertificate    string
+	TLSCertificateKey string
+	MassaNodeServer   string
+	Version           bool
+}
 
-	flag.Parse()
+func parseFlags(server *restapi.Server, startFlags StartServerFlags) {
```

Comment:
I think it's confusing to have 2 parseFlags function, we should name them differently

Reply:
i guess it is more something about configuring the server rather than parsing flag